week-4/implement/count.test.js

```diff
@@ -15,3 +15,29 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+function countChar(char, str){
+
+    let count = 0;
+    let index = 0;
+
+    while (index < str.length){
```

Comment:
Good work. I do want to see you stopping  defining index outside of the loop though. I would also have a look at diffrent types of loops - a for ... in loop would be a better and easier one to use here. https://buzzcoder.gitbooks.io/codecraft-javascript/content/string/loop-through-a-string.html